### ISSUE TITLE
bad pytorch-lightning version

### ISSUE THREAD

**Issue by dsakurai** [rafcc/pytorch-bsf#101]
**Describe the bug**
pip installs an incompatible version of pytorch-lightning. The expected version is pytorch-lightning 1.7.7, although pip gives me 2.0.7. This crashes pytorch-bsf as follows:
```
  File "/Users/Shared/work/shared/code-dev/subdivide-elasticnet/venv/lib/python3.10/site-packages/torch_bsf/bezier_simplex.py", line 397, in fit
    trainer = pl.Trainer(
  File "/Users/Shared/work/shared/code-dev/subdivide-elasticnet/venv/lib/python3.10/site-packages/pytorch_lightning/utilities/argparse.py", line 70, in insert_env_defaults
    return fn(self, **kwargs)
TypeError: Trainer.__init__() got an unexpected keyword argument 'auto_select_gpus'
```

`auto_select_gpus` is an old option abandoned by recent versions of pytorch-lightning.

**To Reproduce**
```
$ pip install pytorch-bsf
...
Using cached pytorch_lightning-2.0.7-py3-none-any.whl
...
```

**Desktop (please complete the following information):**
 - OS: macOS
 - Version 13.0.1
 - Chip: Apple M1 Max

**Comment by NaokiHamada** [comment]
Fixed by [27240ec](https://github.com/rafcc/pytorch-bsf/commit/27240ec6816d2a3e89622d689e945f4561d2d35e)

**Comment by NaokiHamada** [comment]
[27240ec](https://github.com/rafcc/pytorch-bsf/commit/27240ec6816d2a3e89622d689e945f4561d2d35e) has a bug
